ibis/expr/operations/core.py

```diff
@@ -85,7 +85,11 @@ def __coerce__(
     # TODO(kszucs): figure out how to represent not named arguments
     @property
```

Comment:
I think we should turn this into `@attribute` in a follow-up since the `.name` property is accessed multiple times during both construction and compilation.